client/src/components/Form/FormElement.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div :class="cls" :tour_id="id">
+        <div v-if="hasError" class="ui-form-error">
+            <span class="fa fa-arrow-down mr-1" />
+            <span class="ui-form-error-text">{{ error }}</span>
+        </div>
+        <div class="ui-form-title">{{ title }}</div>
+        <div class="ui-form-field">
+            <FormInput :id="id" :value="value" :area="area" @onChange="onChange" />
+            <span class="ui-form-info form-text text-muted mt-2">{{ help }}</span>
+        </div>
+    </div>
+</template>
+<script>
+import FormInput from "./FormInput";
+
+export default {
+    components: {
+        FormInput,
+    },
+    props: {
+        id: {
+            type: String,
+            required: true,
+        },
+        value: {
+            type: String,
+            default: "",
+        },
+        title: {
+            type: String,
+            default: "",
+        },
+        help: {
+            type: String,
+            default: "",
+        },
+        area: {
+            type: Boolean,
+            default: false,
+        },
+        error: {
+            type: String,
+            default: null,
+        },
+    },
+    computed: {
+        hasError() {
+            return !!this.error;
+        },
+        cls() {
+            if (this.hasError) {
+                return "ui-form-element section-row alert alert-danger";
+            } else {
+                return "ui-form-element section-row";
+            }
+        },
+    },
+    methods: {
+        onChange(value) {
```

Comment:
Looks like you are emitting a change event.

Are you familiar with either v-model or .sync syntax for Vue? 

Specifically, if you are writing a FormElement whose job is to update an external variable in a reactive way, emitting events is a good start, but if you conform to some Vue standards the usability of your components becomes a lot better. 

For example:

```html
<FormElement v-model="foo" />

// is shorthand for

<FormElement :value="foo" @input="(val) => foo = val" />
```
This achieved by defining your component to accept an input parameter named "value" and emit an event named "input";

If you have more than one similarly updatable value you can use the .sync property modifier, ex:
```html
<SomeComponent :foo.sync="shoeSize" :bar.sync="hatColor" />

// is the same as

<SomeComponent 
       :foo="shoeSize" @update:foo="(val) => shoeSize = val" 
       :bar="hatColor" @update:bar="(val) => hatColor = val"
/>
```
You can achieve this behavior by using events named "update:foo" and "update:bar".